redocly.yaml

```
@@ -8,4 +8,6 @@ rules:
   no-invalid-media-type-examples: off
   no-path-trailing-slash: off
   operation-2xx-response: off
-  spec-strict-refs: error
\ No newline at end of file
+  spec-strict-refs: error
+  # $ref in examples are not spec-compliant so we perform this check in a python script.
```

Even though the script name may change, I find it useful to link so one can follow the chain of logic.

```suggestion
  # $ref in examples are not spec-compliant, yet we find them useful. Disable this rule
  # and instead check example schema using `check-event-schema-examples.py`
```